crates/libtiny_tui/src/tui.rs

```diff
@@ -377,6 +384,38 @@ impl TUI {
             ret
         };
 
+        // Get tab configs for the type of tab being created
+        let TabConfig { ignore, notifier } = if let Some(config) = self.load_config() {
```

Comment:
This will read and parse the config file every time we crate a tab, right? Why not just store the config somewhere and only reload when asked with `/reload`?

Reply:
Yeah, that does make more sense. Should I add that in this PR?

Reply:
I'd prefer doing it in this PR, yes. Otherwise master will have this issue for a while. It will be best if we never introduce it in the first place.